eng/Versions.props

```
@@ -29,7 +29,7 @@
       It's also used in root Directory.Build.targets to determine the version of the last-built targeting pack.
     -->
     <TargetingPackVersionPrefix>$(VersionPrefix)</TargetingPackVersionPrefix>
-    <ExperimentalVersionPrefix>0.3.$(AspNetCorePatchVersion)</ExperimentalVersionPrefix>
+    <ExperimentalVersionPrefix>0.8.$(AspNetCorePatchVersion)</ExperimentalVersionPrefix>
```

nit: Thoughts on the following so we never need to think about this again❔
```suggestion
    <ExperimentalVersionPrefix>0.$(AspNetCoreMajorVersion).$(AspNetCorePatchVersion)</ExperimentalVersionPrefix>
```
Fortunately, I don't see anything like a `10.1.0` release in our future.

---

I love it.